resources/carrier/en/380.txt

```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2013 The Libphonenumber Authors
+# Copyright (C) 2018 The Libphonenumber Authors
```

Comment:
Please revert to 2013. It is the initial creation of the file. We do not change the year every time we edit the file :)

Reply:
This  year only creation?

Reply:
No, this file was there before only, it was created back in 2013 for the first time. You are now just making few changes :)